prow/initupload/run.go

```diff
@@ -61,6 +54,13 @@ func specToStarted(spec *downwardapi.JobSpec, cloneRecords []clone.Record) gcs.S
 		started.Repos[ref.Org+"/"+ref.Repo] = ref.String()
 	}
 
+	if mainRefs := spec.MainRefs(); mainRefs != nil {
+		started.Repos["RepoVersion"] = shaForRefs(*mainRefs, cloneRecords)
+		if started.Repos["RepoVersion"] == "" {
```

Comment:
This should be outside the `mainRefs` conditional to match the previous behavior. Currently this will always be false.

Reply:
Thanks Cole.